check

```diff
@@ -281,36 +281,26 @@ validate_args() {
   fi
 
   if [[ "${CLI_ARGS_CNT}" -lt 1 ]]; then
-    usage "given less then 1 argument" "1"
+    usage "given less then 1 mandatory argument" "1"
   fi
-  
-  DBURI="None"
 
   for i in $(seq 0 ${CLI_ARGS_POSSIBLE}); do
     if [[ ! -z "${ARG_IS_GIVEN[$i]+x}" ]]; then
       # generate dynamic variables like "$HOST" and export them
       # to make visible from healthcheck scripts
-      eval "${INTERNAL_NAME[$i]}"="${ARG_VALUE[$i]}"
+      eval "export ${INTERNAL_NAME[$i]}"="${ARG_VALUE[$i]}"
       validate_arg_type "${FULL_NAME[$i]}" "${ARG_TYPE[$i]}" "${ARG_VALUE[$i]}"
     else
-      declare -x "${INTERNAL_NAME[$i]}=None"
+      export "${INTERNAL_NAME[$i]}=None"
     fi
   done
 
-  # override --host, --port and --dbname by --db-uri
-  # and set connection options for a psql command
-  if [[ "${DBURI}" = "None" ]]; then
-    PSQL_CONN_OPTIONS="--host=${HOST} --port=${PORT} --dbname=${DBNAME} -U ${USERNAME}"
-  else
-    PSQL_CONN_OPTIONS="'$DBURI'"
-    # threat arguments as given if we've got --db-uri (hack)
-    for i in $(seq 0 ${CLI_ARGS_POSSIBLE}); do
-      [[ "${INTERNAL_NAME[$i]}" = "HOST" ]] && ARG_IS_GIVEN[$i]="true"
-      [[ "${INTERNAL_NAME[$i]}" = "PORT" ]] && ARG_IS_GIVEN[$i]="true"
-      [[ "${INTERNAL_NAME[$i]}" = "DBNAME" ]] && ARG_IS_GIVEN[$i]="true"
-    done
-  fi
+  # fill default (not given) psql connection related variables
+  [[ "${PORT}" = "None" ]] && export PORT=5432
+  [[ "${DBNAME}" = "None" ]] && export DBNAME=postgres
+  [[ "${USERNAME}" = "None" ]] && export USERNAME="${USER}"
 
+  PSQL_CONN_OPTIONS="--host=${HOST} --port=${PORT} --dbname=${DBNAME} --username ${USERNAME}"
```

Comment:
for first 3 options here `=` is used, for the last one it's not.
let's change it, if it's not by purpose?
(`--username=${USERNAME}`)